example/pointed.red

```diff
@@ -46,4 +46,22 @@ let pf (pA : ptype) : pequiv (p→ pbool pA) pA =
         , λ j → bwdfwd/pt i j
         )
   in
-  (fwd, Iso/Equiv _ _ (fwd.0, bwd, λ a i → a, bwdfwd) .1)
+  (fwd, (Iso/Equiv _ _ (fwd.0, bwd, λ a i → a, bwdfwd).1))
```

Comment:
There are some extra parentheses that I do not understand.

Reply:
and let's put the space back before `.1`